src/scripts/settings/userSettings.js

```diff
@@ -265,3 +265,33 @@ import events from 'events';
     }
 
 /* eslint-enable indent */
+export default {
```

Comment:
I tried to turn off fast fadein, but `userSettings` is broken when imported at ES6 module.

Reply:
That's weird, aside from Playback (which is fixed in another PR), user settings work perfectly for me.

Though I would like to set the default limit to 100 (because of the lack of infinite scrolling) and disable the fast fade-in by default.

Reply:
Maybe we are using two different instances of `userSerrings`: `userSerrings` (imported in non-ES6) and `userSerrings.default` (imported in ES6).
I am not sure, but we probably have something like this
```js
{
    setUserInfo: setUserInfo,
    getData: getData,
    importFrom: importFrom,
    set: set,
    get: get,
    ...
    default: {
        setUserInfo: setUserInfo,
        getData: getData,
        importFrom: importFrom,
        set: set,
        get: get,
        ...
    }
}
```

Reply:
here `this.currentUserId` is `undefined` when accessed from `imageLoader` and contains valid id when accessed from user settings.
https://github.com/jellyfin/jellyfin-web/blob/2cb5e8c5d07e4954f608ec1fb4bd3fbb62909a42/src/scripts/settings/userSettings.js#L67

Reply:
Should be fixed with b8422a2